frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -397,11 +397,11 @@ const EnvFromPairElement = DragSource(DRAGGABLE_TYPE.ENV_FROM_ROW, pairSource, c
     return connectDropTarget(
       connectDragPreview(
         <div className={classNames('row', isDragging ? 'pairs-list__row-dragging' : 'pairs-list__row')} ref={node => this.node = node}>
-          <div className="col-xs-1 pairs-list__action-icon">
-            { !readOnly &&
-              <React.Fragment>{connectDragSource(<i className="pficon pficon-drag-drop pairs-list__action-icon--reorder" />)}</React.Fragment>
-            }
-          </div>
+          { !readOnly &&
```

Comment:
I'd think you would want to check allowSorting here too? @jcaianirh can you confirm?

Seems like you could skip all of `connectDropTarget` if we're not supporting drag n drop.

Reply:
@spadgett, @rebeccaalpert Because EnvFromPairElement is created only for the purposes of the env editor with sorting capability we don't have to check for allowSorting here. We have to check for that in the PairElement component because thats also used for non sorted parent components such as the annotation editor. If